devito/passes/clusters/blocking.py

```diff
@@ -88,10 +104,14 @@ def callback(self, clusters, prefix):
                 exprs = [uxreplace(e, {d: bd}) for e in c.exprs]
 
                 # The new Cluster properties
+                # TILABLE property is dropped after the blocking.
+                # SKEWABLE is dropped as well, but only from the new
+                # block dimensions.
                 properties = dict(c.properties)
                 properties.pop(d)
                 properties.update({bd: c.properties[d] - {TILABLE} for bd in block_dims})
-
+                properties.update({bd: c.properties[d] - {SKEWABLE}
```

Comment:
replace `bd` with `i` if it makes it fit on a single line?

Reply:
if so, change `bd` also in the line above

Reply:
94>90 :-(

Reply:
can fit if you also drop the spaces between the minus (both lines)

Reply:
Unfortunately still doesn;t fit.